zhaquirks/lixee/zlinky.py

```diff
@@ -186,3 +195,15 @@ class ZLinkyTIC(CustomDevice):
             },
         },
     }
+
+
+class ZLinkyTICFWV12(ZLinkyTIC):
+    """ZLinky_TIC from LiXee with firmware v12.0+."""
+
+    signature = deepcopy(ZLinkyTIC.signature)
```

Comment:
Should I keep the same model name so that the fix in https://github.com/home-assistant/core/pull/82602/files#diff-97adf01229931f17610655838232182c16a7f10291291d780cfd472be187406eR510 continues to work?